src/components/Forms/ChangeUserData/index.tsx

```diff
@@ -16,13 +16,15 @@ const ChangeUserData = () => {
   const theme = useTheme();
   const matches = useMediaQuery(theme.breakpoints.down('xs'));
   const { userInfo, setAuthInfo, logout } = useAuthContextState();
-  const [ChangeUserData, { error, data, loading, called }] =
-    useMutation<QueryData, QueryVars>(CHANGE_USER_DATA);
+  const [ChangeUserData, { error, data, loading, called }] = useMutation<QueryData, QueryVars>(
+    CHANGE_USER_DATA
+  );
 
   return (
     <Formik
       initialValues={{ ..._.omit(userInfo, '_id') }}
       onSubmit={async (values) => {
+        console.log(values);
```

Comment:
Creep something 🙂

Reply:
I was testing something and forgot to remove 😄